app/components/goodcity/orders-package-block.js

```diff
@@ -147,6 +163,9 @@ export default Ember.Component.extend(AsyncMixin, {
       await this.runTask(
         this.get("actionRunner").execAction(ordersPkg, actionName, params)
       );
+
+      // fire & forget
+      this.get("store").findRecord("item", itemId, { reload: true });
```

Comment:
Is this for updating an item?
If yes, are we not sending live updates for `orders_packages`? 🤔

Reply:
Yeah Steve and I found a weird edge case. Live updates are ignored when they are triggered by the session itself (for good reasons, apps would break because live updates would land before the requests finishes).

So it was either a background reload, or we fiddle/hack the subscriptions to let only some updates through. None of which is really great, we went for the lesser of two evils

Reply:
BTW, this is for a deletion rather than an update. The API has no way of letting the front-end know that a record was deleted, apart from live updates or reloading.

Reply:
I think `unloadRecord` would be a better choice here.
https://api.emberjs.com/ember-data/3.10/classes/DS.Model/methods/unloadRecord?anchor=unloadRecord

Reply:
We don't want to unload anything per-say. We just want the record in the same state as the backend, one of it's relationships may or may not have been deleted.
The JSON we receive from the backend is a snapshopt of the relationshipts, but ember just adds what it receives to the store, without doing a diff on what currently exists.
So we either refresh the record, or we let the live-updates through.